### ISSUE TITLE
Randomly becomes unable to connect, needs restart

### ISSUE THREAD

**Issue by php4fan** [telegramdesktop/tdesktop#27414]
### Steps to reproduce

This has already happened to me twice since I've started using the Flatpak version of Telegram Desktop

1. Use Telegram Desktop
2. Leave the computer unattended for some time (this may cause it to suspend)
3. come back, and give focus to telegram again

### Expected behaviour

should be usable

### Actual behaviour

No matter what chat I try to open, if I had opened the same chat before, it shows the contents but not up to date; if it's a chat that I hadn't opened before, even old content is not available.

At the  top it says "Connecting..." with a spinning icon, and stays like that forever.

Basically it's failing to connect, there is no way to make it retry, it doesn't retry (or it does and keeps failing) and it is unusable.

Needless to say the network is fine.

The only way to get Telegram working again is to restart it.

### Operating system

Manjaro Linux

### Version of Telegram Desktop

4.14.9

### Installation source

Flatpak

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/16093
